wallet/wallet.go

```diff
@@ -2382,6 +2382,88 @@ func (w *Wallet) TransactionSummary(txHash *chainhash.Hash) (*TransactionSummary
 	return txSummary, err
 }
 
+// GetTicketsResult response struct for gettickets rpc request
+type GetTicketsResult struct {
+	Tickets []*TicketSummary
+}
+
+// GetTickets implements the rpc request command for gettickets
+func (w *Wallet) GetTickets(startBlock, endBlock *BlockIdentifier, cancel <-chan struct{}) (*GetTicketsResult, error) {
```

Comment:
I would change this function signature to something more like:

```
func (w *Wallet) GetTickets(ctx context.Context, startBlock, endBlock *BlockIdentifier, results chan<- *TicketSummary) error
```

A context should be used for cancellation and should be the first parameter.  The channel is used to stream the results back to the caller rather than returning all of the results at once.  This will allow the RPC server to actually stream the results to clients as it creates them, instead of loading all of the data up at once into a single slice and then sending messages to the client stream.  (This is also a problem with the GetTransactions RPC and should be resolved in a separate commit)